stack.yaml

```diff
@@ -94,8 +94,8 @@ packages:
 # https://github.com/haskell-distributed/network-transport-tcp/compare/master...serokell:3d56652123bd296dc759cd31947eb2a17924e68a?expand=1
 # As above, seems relatively straightforward to PR.
 - location:
-    git: https://github.com/avieth/network-transport-tcp
-    commit: 2634e5e32178bb0456d800d133f8664321daa2ef # csl-0.6.0
+    git: https://github.com/deepfire/network-transport-tcp
+    commit: 44f84a887cf954dd924f28b7e8da107228e57c13 # csl-0.6.0
```

Comment:
> change in network-transport-tcp to be more lenient regarding remote address claims

In `44f84a887cf954dd924f28b7e8da107228e57c13`, is there good reason for dropping the `theirHost == numericHost || theirHost == resolvedHost` check?

Reply:
It was giving problems when starting a bunch of nodes on the same machine -- and I went to extra lengths to try avoiding this:

1. setting up a VDE2 [1] virtual LAN, with one TAP interface per node (and hence a dedicated non-loopback IP)
2. Implementing source policy routing [2]

This got me quite far regarding the address inconsistencies that that error check was giving me, but alas, I still couldn't get past.

There is a suspicion that it probably wasn't critical anyway -- but perhaps we want to double-check with @avieth.

--
1. https://github.com/virtualsquare/vde-2
2.  https://www.tldp.org/HOWTO/Adv-Routing-HOWTO/lartc.rpdb.simple.html

Reply:
Dropped the `network-transport-tcp` changes & replaced them with a CLI flag driving the parameter that @avieth pointed out.